### PR TITLE
Feature/394 Time Dependent Inversion Solution

### LINKED ISSUE #395: Add TDIS page with the link `TimeDependentSolution/${id}`
When viewing a TimeDependentInversionSolution in TUI I want to see the information and meta data.

Note a TimeDependentInversionSolution looks almost identical to an ScaledInversionSolution. 
 - it has a source_solution attribute returns the IS from which SIS was derived
 - in meta you'll see time dependent kv items

**Done When:**

 - [x] Have Time Dependent Inversion Solution page working at link /TimeDependentInversionSolution/${id}
 - [x] User can view Solution Diagnostic view
 - [x] Add this link to App.tsx routing
 - [x] Add this link to AutomationTask file table ??

### PR DESCRIPTION
**Pull request by benjamineac**
Adds /TimeDependentInversionSolution/ page, GT flipchart support for TDIS and find by id support.

closes #395 and #396

### CODE REVIEW

**Review by chrisdicaprio — COMMENTED** [review]
@benjamineac NAMED FAULTS is not working in the flipchart

**Review by chrisdicaprio — APPROVED** [review]

**Review by chrisbc — APPROVED** [review]

**Comment by chrisbc** [comment]
@benjamineac I added @0qy to reviewers list. Two things I thought of ....

File: src/components/SearchResult.tsx [hunk 1]
```diff
@@ -19,8 +19,14 @@ const SearchResult: React.FC<SearchResultProps> = ({ queryRef }: SearchResultPro
       <Box>
         {data?.search?.search_result?.edges?.map((edge) => {
           const searchType = edge?.node?.__typename;
+          console.log(searchType);
```
Comment (0qy) [inline src/components/SearchResult.tsx:22]: delete console.log

File: src/components/mySolutions/MySolutionsListItem.tsx [hunk 2]
```diff
@@ -23,25 +23,34 @@ interface MySolutionsListItemProps {
 }
 const MySolutionsListItem: React.FC<MySolutionsListItemProps> = ({ automationTask }: MySolutionsListItemProps) => {
   const parentTask = automationTask?.parents?.edges[0]?.node?.parent;
-  const inversionSolution = automationTask?.inversion_solution;
-  const scaledInversionSolutionFile = automationTask?.files?.edges.filter((file) => file?.node?.file?.source_solution);
-  const scaledInversionSolution = scaledInversionSolutionFile && scaledInversionSolutionFile[0]?.node?.file;
 
+  console.log(automationTask);
```
Comment (0qy) [inline src/components/mySolutions/MySolutionsListItem.tsx:27]: delete console.log

File: src/components/mySolutions/MySolutionsListItem.tsx [hunk 3]
```diff
@@ -23,25 +23,34 @@ interface MySolutionsListItemProps {
...
-              }`}
-            >
-              [more]
-            </Link>
+            {() => {
+              switch (automationTask?.task_type) {
+                case 'TIME_DEPENDENT_SOLUTION':
+                  <Link to={`/TimeDependentSolution/${automationTask?.id}`}>[more]</Link>;
+                  break;
+                case 'INVERSION':
```
Comment (0qy) [inline src/components/mySolutions/MySolutionsListItem.tsx:46]: Shouldn't this be 'INVERSION_SOLUTION'?

File: @types/decs.d.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+declare module '@deck.gl/react';
+declare module 'react-map-gl';
+declare module '@deck.gl/layers';
+declare module '@deck.gl/mapbox';
+declare module '@deck.gl/core';
```
Comment (0qy) [inline @types/decs.d.ts:5]: Are these relevant to this feature?

**Review by 0qy — APPROVED** [review]
